Library/Formula/notmuch.rb

```diff
@@ -38,8 +40,17 @@ def install
     else
       args << "--without-emacs"
     end
-    system "./configure", *args
 
+    system "./configure", *args
     system "make", "V=1", "install"
+
+    if build.with? "python"
```

Comment:
One last thing, and it's my fault, sorry: can you remove this conditional since it's implied by `each_python`?